include/boost/geometry/algorithms/detail/distance/multipoint_to_geometry.hpp

```diff
@@ -99,6 +101,18 @@ struct multipoint_to_linear
                      boost::end(multipoint),
                      linear,
                      strategy);
+
+        bool is_multi = boost::is_same
+                <
+                    typename tag<Linear>::type,
+                    multi_linestring_tag
+                >::type::value;
```

Comment:
It's rather `is_multi_linestring` not `is_multi` which could be interpreted also as `multi_point` and `multi_polygon`.

Reply:
right, thanks